corehq/apps/toggle_ui/views.py

```diff
@@ -341,19 +341,30 @@ def _get_service_type(toggle):
     """Returns subscription service type for each toggle
     """
     service_type = {}
+    nested = defaultdict(lambda: defaultdict(list))
+    paused = defaultdict(list)
     for enabled in toggle.enabled_users:
         if _namespace_domain(enabled):
             domain = _enabled_item_name(enabled)
             if subscription := Subscription.get_active_subscription_by_domain(domain):
                 service_type[domain] = f"{subscription.service_type} : {subscription.plan_version.plan.name}"
+                if subscription.plan_version.plan.name.startswith("CommCare Paused Edition"):
```

Comment:
Nit: looking at https://www.commcarehq.org/hq/flags/edit/user_reports/?usage_info=true I see both "PRODUCT : CommCare Paused Edition" and "PRODUCT : CommCare Paused Plan".  However, looking in the accounting code, it looks like there's a `SoftwarePlanVersion.is_paused` property that you could check instead, which might be more reliable.  Looks like that's what's used in accounting code to determine whether something is paused.